packages/request-node/src/requestNode.ts

```diff
@@ -133,6 +134,9 @@ export class RequestNode {
     // Enable all CORS requests
     this.express.use(cors());
 
+    // Enable logging of all requests
+    this.express.use(morgan('combined'));
```

Comment:
Nice :+1: 

Part of me worries this makes some of the other log statements redundant, but it's not a large enough concern to warrant a change.